_build/templates/default/package.json

```diff
@@ -1,31 +1,40 @@
 {
-  "name": "revolution-theme-default",
-  "title": "MODX Revolution",
-  "version": "3.0.0",
-  "license": "GPL-2.0+",
-  "repository": "https://github.com/modxcms/revolution",
-  "devDependencies": {
-    "grunt": "^1.3.0",
-    "grunt-autoprefixer": "^3.0.4",
-    "grunt-contrib-concat": "^1.0.1",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-csslint": "^2.0.0",
-    "grunt-contrib-cssmin": "^3.0.0",
-    "grunt-contrib-uglify": "^4.0.1",
-    "grunt-contrib-watch": "^1.1.0",
-    "grunt-imageoptim": "^1.4.4",
-    "grunt-notify": "^0.4.5",
-    "grunt-sass": "^3.1.0",
-    "node": "^14.15.0",
-    "node-sass": "^5.0.0"
-  },
-  "dependencies": {
-    "bourbon": "^5.1.0",
-    "bourbon-neat": "^3.0.0",
-    "@fortawesome/fontawesome-free": "^5.7.0",
-    "normalize-scss": "^7.0.1"
-  },
-  "scripts": {
-    "build": "grunt build"
-  }
+    "name": "revolution-theme-default",
+    "title": "MODX Revolution",
+    "version": "3.0.0",
+    "license": "GPL-2.0+",
+    "repository": "https://github.com/modxcms/revolution",
+    "devDependencies": {
+        "@lodder/grunt-postcss": "^3.0.1",
+        "autoprefixer": "^10.3.3",
+        "bourbon": "^5.1.0",
+        "bourbon-neat": "^3.0.0",
+        "cssnano": "^5.0.8",
+        "grunt": "^1.4.1",
+        "grunt-contrib-concat": "^1.0.1",
+        "grunt-contrib-copy": "^1.0.0",
+        "grunt-contrib-watch": "^1.1.0",
+        "grunt-dart-sass": "^2.0.1",
+        "grunt-notify": "^0.4.5",
+        "grunt-terser": "^2.0.0",
+        "node": "^14.15.0",
+        "normalize-scss": "^7.0.1",
+        "postcss": "^8.3.6"
```

Comment:
Why you use postcss and sass both?

Reply:
postcss is being used as a base for cssnano and prefixer. I'm not using the postcss scss plugin to parse scss because it's not entirely standard sass.